modules/mod_cck_breadcrumbs/tmpl/microdata.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+* @version 			SEBLOD 3.x Core
+* @package			SEBLOD (App Builder & CCK) // SEBLOD nano (Form Builder)
+* @url				http://www.seblod.com
+* @editor			Octopoos - www.octopoos.com
+* @copyright		Copyright (C) 2005 - 2012 Open Source Matters, Inc. All rights reserved.
+* @license 			GNU General Public License version 2 or later; see _LICENSE.php
+**/
+
+defined( '_JEXEC' ) or die;
+?>
+<div class="cck_module_breadcrumbs <?php echo $class_sfx; ?>">
+	<ul class="list-inline" itemscope itemtype="http://schema.org/BreadcrumbList">
+	<?php
+	$show_last	=	$params->get( 'showLast', 1 );
+	if ( $params->get( 'showHere', 1 ) ) {
+			echo '<span class="showHere">'.JText::_( 'MOD_CCK_BREADCRUMBS_HERE' ).'</span>';
+	}
+	for ( $i = 0; $i < $count; $i++ ) {
+		echo '<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">';
+		if ( $i < $count -1 ) { // If not the last item in the breadcrumbs add the separator
+			if ( !empty($list[$i]->link ) ) {
+				echo '<a itemscope itemtype="http://schema.org/Thing"
```

Comment:
you need to remove 
`itemscope itemtype="http://schema.org/Thing" `
part from a and span elements, otherwise google does not recognize it correctly (it does not read url property)